source/ways-of-working/new-component/gitops-flux.html.md.erb

```diff
@@ -42,9 +42,9 @@ One of our principles being <b>Everything as Code</b>, we practise [Gitops](http
 
 - For issue like latest image not updated to git or a HelmRelease  CRD not updated/created in cluster , you need to look at flux pod logs. 
 
-    `kubectl logs -n admin -l app=flux --tail=100`
+    `kubectl logs -n admin -l app=flux --tail=100 | grep <Release Name>`
```

Comment:
somewhat wondering if this should be another line, i.e. you might not want to filter it?